<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1049</width>
    <height>725</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Plazza</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>20</y>
      <width>20</width>
      <height>641</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="line2">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>260</y>
      <width>791</width>
      <height>21</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line3">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>30</y>
      <width>20</width>
      <height>231</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>20</y>
      <width>381</width>
      <height>231</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="newOrderLayout">
     <item row="2" column="0">
      <widget class="QComboBox" name="boxPizzaType">
       <item>
        <property name="text">
         <string>Regina</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Margarita</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Americana</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Fantasia</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QComboBox" name="boxPizzaSize">
       <item>
        <property name="text">
         <string>S</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>M</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>L</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>XL</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>XXL</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLCDNumber" name="pizzaNumber">
       <property name="smallDecimalPoint">
        <bool>false</bool>
       </property>
       <property name="intValue" stdset="0">
        <number>1</number>
       </property>
      </widget>
     </item>
     <item row="4" column="2">
      <widget class="QLabel" name="numberOfPizzas">
       <property name="text">
        <string>Number of Pizzas</string>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QLabel" name="pizzaSize">
       <property name="text">
        <string>Pizza Size</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QLabel" name="pizzaType">
       <property name="text">
        <string>Pizza Type</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QSlider" name="sliderPizzaNumber">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="oldSchool">
       <property name="text">
        <string>Old School Command Line:</string>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QPushButton" name="buttonOrder">
       <property name="text">
        <string>CHIEF, MY ORDER !</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="newOrder">
       <property name="text">
        <string>New Order:</string>
       </property>
      </widget>
     </item>
     <item row="8" column="0" colspan="3">
      <widget class="QLineEdit" name="linePizzaOrder"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>20</y>
      <width>371</width>
      <height>231</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="stockLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="stockOfIngredients">
       <property name="text">
        <string>Stock of Ingredients: </string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QTableWidget" name="tableStock">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="minimumSize">
        <size>
         <width>369</width>
         <height>206</height>
        </size>
       </property>
       <property name="editTriggers">
        <set>QAbstractItemView::NoEditTriggers</set>
       </property>
       <attribute name="horizontalHeaderDefaultSectionSize">
        <number>220</number>
       </attribute>
       <attribute name="horizontalHeaderMinimumSectionSize">
        <number>100</number>
       </attribute>
       <attribute name="horizontalHeaderStretchLastSection">
        <bool>true</bool>
       </attribute>
       <column>
        <property name="text">
         <string>Ingredient</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Stock</string>
        </property>
       </column>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>280</y>
      <width>791</width>
      <height>381</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="ordersLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="awaitingPizzas">
       <property name="text">
        <string>Awaiting pizzas:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="finishedPizzas">
       <property name="text">
        <string>Finished pizzas :</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="processingPizzas">
       <property name="text">
        <string>Processing pizzas :</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QTableWidget" name="tableWaiting">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="editTriggers">
        <set>QAbstractItemView::NoEditTriggers</set>
       </property>
       <attribute name="horizontalHeaderDefaultSectionSize">
        <number>150</number>
       </attribute>
       <attribute name="horizontalHeaderStretchLastSection">
        <bool>true</bool>
       </attribute>
       <column>
        <property name="text">
         <string>Type</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Size</string>
        </property>
       </column>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QTableWidget" name="tableProcessing">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="editTriggers">
        <set>QAbstractItemView::NoEditTriggers</set>
       </property>
       <attribute name="horizontalHeaderDefaultSectionSize">
        <number>150</number>
       </attribute>
       <attribute name="horizontalHeaderStretchLastSection">
        <bool>true</bool>
       </attribute>
       <column>
        <property name="text">
         <string>Type</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Size</string>
        </property>
       </column>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QTableWidget" name="tableFinished">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="editTriggers">
        <set>QAbstractItemView::NoEditTriggers</set>
       </property>
       <attribute name="horizontalHeaderDefaultSectionSize">
        <number>150</number>
       </attribute>
       <attribute name="horizontalHeaderStretchLastSection">
        <bool>true</bool>
       </attribute>
       <column>
        <property name="text">
         <string>Type</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Size</string>
        </property>
       </column>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>221</width>
      <height>641</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="kitchensLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="kitchenList">
       <property name="text">
        <string>Kitchen List: </string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QListWidget" name="tableKitchen"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>670</y>
      <width>1031</width>
      <height>27</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="infoLayout">
     <item>
      <widget class="QLabel" name="labelWallet">
       <property name="maximumSize">
        <size>
         <width>200</width>
         <height>100</height>
        </size>
       </property>
       <property name="text">
        <string>Your wallet: $10.000</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QProgressBar" name="barAchievment">
       <property name="maximumSize">
        <size>
         <width>200</width>
         <height>100</height>
        </size>
       </property>
       <property name="value">
        <number>1</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelInfos">
       <property name="text">
        <string>Last informations: </string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuPlazza">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1049</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMainWindow">
    <property name="title">
     <string>Plazza</string>
    </property>
    <addaction name="actionAchievment"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionOpenMan"/>
    <addaction name="actionOpenQt"/>
    <addaction name="actionOpenLog"/>
   </widget>
   <addaction name="menuHelp"/>
   <addaction name="menuMainWindow"/>
  </widget>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
   <property name="shortcut">
    <string>Esc</string>
   </property>
  </action>
  <action name="actionNew_command">
   <property name="text">
    <string>New command</string>
   </property>
  </action>
  <action name="actionOpenMan">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>About Plazza</string>
   </property>
   <property name="shortcut">
    <string>F1</string>
   </property>
   <property name="visible">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionOpenLog">
   <property name="text">
    <string>Open Logs</string>
   </property>
   <property name="shortcut">
    <string>F3</string>
   </property>
  </action>
  <action name="actionAchievment">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Reach $1.000.000 to unlock</string>
   </property>
  </action>
  <action name="actionOpenQt">
   <property name="text">
    <string>About Qt</string>
   </property>
   <property name="shortcut">
    <string>F2</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
